Project.cmake

```diff
@@ -36,6 +40,9 @@ if(WIN32)
 
 	# Install binaries in the root of the install directory
 	set(INSTALL_BIN_DIR "/" CACHE PATH "Where to place binaries in the instalation")
+elseif(APPLE)
+	set(INSTALL_BIN_DIR "Contents/MacOS" CACHE PATH "Where to place binaries in the instalation")
+	set(INSTALL_ASSET_DIR "Resources")
```

Comment:
Does this work? It looks like the bundle generator adds "Contents/" to the package automatically. What tree do you get when you mount the bundle with `INSTALL_BIN_DIR` just set to "MacOS"?